packages/chart/src/LazyChart.tsx

```diff
@@ -0,0 +1,15 @@
+import { LoadingOverlay } from '@deephaven/components';
+import { lazy, Suspense } from 'react';
+
+const Chart = lazy(() => import('./Chart.js'));
```

Comment:
I don't think you need the `.js` here.
```suggestion
const Chart = lazy(() => import('./Chart'));
```

Reply:
I'll check if our babel add extensions step adds it to dynamic imports. Adding the `.js` is proper ESM

Reply:
Our babel plugin does not add extensions to dynamic imports, so I'm going to leave the `.js` so it's valid ESM